Documentation/LayoutCodeBehind.md

```diff
@@ -0,0 +1,67 @@
+---
+id: 11763499-79e9-4868-83e6-41f3061745d1
+title: "Layout CodeBehind"
+dateupdated: 2018-01-29
+---
+
+<a name="Overview" class="injected"></a>
+
+# Overview
+
+Xamarin.Android supports the auto generation of "Code Behind" classes. These classes
+can reduce the amount code a developer writes. You can end up replacing code like
+
+        var button = FindViewById<Button> (Resource.Id.myButton);
+        button.Click += delegate {
+        };
+
+with
+
+        myButton.Click += delegate {
```

Comment:
`code` indentation is inconsistent within the file. Here, it's 8 spaces, while later below it's 4 spaces.

Indentation should be consistent.